pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1487,7 +1487,7 @@ func checkIfDiskReadyToUseFunc(filename string) (bool, error) {
 		return true, nil
 	}
 	// Before attempting to attach, ensure we can open the file
-	file, err := os.OpenFile(filename, os.O_RDWR, 0660)
+	file, err := os.OpenFile(filename, os.O_RDWR, 0600)
```

Comment:
Same as above.

Reply:
> Same as above.

Restricting the RDWR permission to be root-only here did not cause trouble for the tests, because the function is in virt-launcher process running as root and the file must have been created (O_RDWR does not create file).